src/components/Charts/BarChart.tsx

```diff
@@ -0,0 +1,97 @@
+import { Bar } from 'react-chartjs-2';
+
+const BarChart = () => {
+  // const data: any = {
+  //   labels: [""],
+  //   datasets: [
+  //     {
+  //       label: ["Male"],
+  //       data: [600],
+  //       backgroundColor: ["rgba(255, 99, 132, 0.6)"],
+  //       borderSkipped: false,
+  //       borderRadius: Number.MAX_VALUE,
+  //       barThickness: 50,
+  //     },
+
+  //     {
+  //       label: ["Female"],
+  //       data: [500],
+  //       backgroundColor: ["rgba(54, 162, 235, 0.6)"],
+
+  //       borderSkipped: false,
+  //       borderRadius: Number.MAX_VALUE,
+  //       barThickness: 50,
+  //     },
+  //     {
+  //       label: ["?"],
+  //       data: [50],
+  //       backgroundColor: ["rgba(255, 206, 86, 0.6)"],
+  //       borderRadius: Number.MAX_VALUE,
+  //       borderSkipped: false,
+  //       barThickness: 50,
+  //     },
+  //   ],
+  // };
+
+  // // const options: any = {
+  // //   scales: {
+  // //     xAxes: [
+  // //       {
+  // //         barThickness: 2, // number (pixels) or 'flex'
+  // //         maxBarThickness: 1, // number (pixels)
+  // //       },
+  // //     ],
+  // //   },
+  // // };
+
+  // // const options: any = {
+  // //   cornerRadius: 4,
+  // // };
```

Comment:
if we are not using this code lets remove it from here entirely

Reply:
- [x] separated the tickets